crates/nargo/tests/test_data/to_be_bytes/src/main.nr

```diff
@@ -0,0 +1,11 @@
+use dep::std;
+
+fn main(x : Field) -> pub [u8; 31] {
+    // The result of this byte array will be big-endian
+    let byte_array = x.to_be_bytes(31);
+    let mut bytes = [0; 31];
+    for i in 0..31 {
+        bytes[i] = byte_array[i];
```

Comment:
could you add some constraints that validate the output is correct based on the value of x (2040124)?